src/data/openings.json

```diff
@@ -145,5 +145,21 @@
       "Expertise in WordPress."
     ],
     "applicationUrl": "https://docs.google.com/forms/d/e/1FAIpQLSf4Mk-vxO3Nl193LkCSGPOrrT5uDvA27peH8du7hSnfUsxb0A/viewform"
+  },
+  {
+    "openingId": "marketing-specialist",
```

Comment:
can u use a different `openingId`? seems to clash with an existing one.

Reply:
sure.